packages/rule-components/src/RuleFilters/ansibleSupport.js

```diff
@@ -0,0 +1,17 @@
+import { ansibleSupportLabels, ansibleSupportValues } from '../RuleTable/constants';
+export default ({ onChange, value, ...props } = { onChange: () => undefined }) => ({
+    ...props,
+    label: 'Ansible support',
+    value: 'ansible_support',
+    type: 'checkbox',
+    filterValues: {
+        value,
+        onChange,
+        items: ansibleSupportValues.map((_item, key) => ({
+            label: ansibleSupportLabels[key],
+            textual: ansibleSupportLabels[key],
+            value: ansibleSupportValues[key]
```

Comment:
Maybe better to use here `_item` instead of `ansibleSupportValues[key]` and in couple of other files too?